client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -41,22 +43,27 @@ class PurchasePlanDetails extends Component {
 			return null;
 		}
 
-		if ( pluginList.length < 1 ) {
-			return null;
-		}
+		const headerText = translate( '%(plan)s Plan', {
+			args: {
+				plan: getName( purchase )
+			}
+		} );
 
 		return (
-			<Card>
-				<QueryPluginKeys siteId={ selectedSite.ID } />
-				{ pluginList.map( ( plugin, i ) => {
-					return (
-						<FormFieldset key={ i }>
-							<FormLabel htmlFor={ `plugin-${ plugin.slug }` }>{ this.renderPluginLabel( plugin.slug ) }</FormLabel>
-							<ClipboardButtonInput id={ `plugin-${ plugin.slug }` } value={ plugin.key } />
-						</FormFieldset>
-					);
-				} ) }
-			</Card>
+			<div>
+				<SectionHeader label={ headerText } />
+				<Card>
+					<QueryPluginKeys siteId={ selectedSite.ID } />
```

Comment:
Nitpicking: I usually try to render the query components as early as possible; does is make sense for this one to be placed above the `<SectionHeader />`?